pkg/sql/execinfrapb/processors_bulk_io.proto

```diff
@@ -145,6 +143,17 @@ message CSVWriterSpec {
   optional roachpb.CSVOptions options = 3 [(gogoproto.nullable) = false];
   // chunk_rows is num rows to write per file. 0 = no limit.
   optional int64 chunk_rows = 4 [(gogoproto.nullable) = false];
+
+  // Compression list of the compression codecs which are currently
+  // supported for CSVWriter spec
+  enum Compression {
+    none = 0;
+    gzip = 1;
+  }
+
+  // compression code type to be used while epxporting CSV file,
+  // if it's null (i.e. not defined), no compression will be used.
```

Comment:
```suggestion
```
The second line of the comment isn't needed: nullable = false actually means it _cannot_ be null (the field type will be `int` not `*int`) so there is no "null" or "undefined" state to reason about, just its the zero value which in this case is None.